hera_cal/abscal.py

```diff
@@ -1446,28 +1450,42 @@ def abscal_run(data_file, model_files, refant=None, calfits_infile=None, verbose
         return return_obj
 
 
-def cut_bls(datacontainer, bls, bl_cut):
+def cut_bls(datacontainer, bls, min_bl_cut=None, max_bl_cut=None, inplace=False):
```

Comment:
Data containers (if they come from a file) now contain `antpos` inside them. Do we want to remove the dependence on `bls`?

Reply:
No?

Reply:
ok